src/keycloak/keycloak.service.ts

```diff
@@ -39,13 +40,18 @@ export interface Token {
   readonly expires_in?: number;
 }
 
+export enum ClientType {
+  ADMIN = 'admin',
+  USER = 'user',
```

Comment:
I would prefer to use a more clear and convenient name

```suggestion
  PUBLIC = 'public',
```